src/metabase/cmd/env_var_dox.clj

```diff
@@ -0,0 +1,110 @@
+(ns metabase.cmd.env-var-dox
+  "Code to generate docs for environment variables. You can generate
+  docs by running: `clojure -M:ee:run environment-variables-documentation`"
+  (:require [clojure.java.classpath :as classpath]
+            [clojure.java.io :as io]
+            [clojure.string :as str]
+            [clojure.tools.namespace.find :as ns.find]
+            [clojure.tools.reader.edn :as edn]
+            [metabase.models.setting :as setting]
+            [metabase.util :as u]))
+
+(def env-vars-not-to-mess-with
+  "Flamber advises that people avoid touching these environment variables."
+  (set (edn/read-string (slurp (io/file "src/metabase/cmd/resources/env-vars-to-avoid.edn")))))
```

Comment:
I should also mention that if you are going to leave this in a file it will only work if you run it with `clj ...` and not with `java -jar`. Maybe that's ok, but the command will still be there in the JAR so we might as well make it work there too.

```suggestion
  (set (edn/read-string (slurp (io/resource "metabase/cmd/resources/env-vars-to-avoid.edn")))))
```

should do the trick.

I'd also suggest that you don't really need to stick this in a `resources/` folder -- putting it in `cmd/` (the same folder as this code source file) is probably fine